src/rdb.c

```diff
@@ -3596,8 +3600,11 @@ int rdbSaveToSlavesSockets(int req, rdbSaveInfo *rsi) {
         /* Close the reading part, so that if the parent crashes, the child will
          * get a write error and exit. */
         close(server.rdb_pipe_read);
-
-        serverSetProcTitle("redis-rdb-to-slaves");
+        if (strstr(server.exec_argv[0],"redis-server") != NULL) {
+            serverSetProcTitle("redis-rdb-to-slaves");
+        } else {
+            serverSetProcTitle("valkey-rdb-to-slaves");
```

Comment:
This is a new name, so let's avoid "slaves". Let's call it "valkey-rdb-to-replicas".

Reply:
Thanks! yeah make sense, I have updated as replicas.